.eslintignore

```diff
@@ -1,2 +1,3 @@
 flow-typed/*
 node_modules/
+templates/
```

Comment:
Shouldn't we lint this folder to make sure we don't introduce regressions? I don't know if we ship with linters/flow yet, but I think we probably should.

Reply:
Ideally I'd like to, but with our linting configuration it complains when a module cannot be resolved (meaning we'd have to "yarn install" the template just to make linting happy ( and I suppose then clean up the files so as to not mess up publishing and what not).